fees/chainlink.ts

```diff
@@ -251,6 +278,14 @@ const adapter: BreakdownAdapter = {
       [CHAIN.POLYGON]: {
         fetch: fetchKeeper(CHAIN.POLYGON),
         start: async ()  => 1675382400,
+      },
+      [CHAIN.FANTOM]: {
+        fetch: fetchKeeper(CHAIN.FANTOM),
+        start: async ()  => 1675382400,
+      },
+      [CHAIN.AVAX]: {
```

Comment:
could you add keepers for arbitrum and optimism too?

Reply:
added, but i can not cal tx fees on optimism then can't cal correct rev

Reply:
leave revenue empty there then